index.ts

```diff
@@ -45,45 +44,6 @@ function updateStatus(status?: string, ...extra: unknown[]) {
 	console.log(status, ...extra);
 }
 
-async function waitForToken() {
```

Comment:
I'll probably restore this after switching to svelte. I think it was used when the first call failed